README.rst

```diff
@@ -45,15 +45,15 @@ The ``-w`` flag will actually implement the changes recommended by codespell. No
 
     codespell -I FILE, --ignore-words=FILE
 
-The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries. ::
+Get a list of words to ignore from _FILE_. The _FILE_ format is one word per line. **Important note:** Words are case-sensitive, based on how they are listed in the codespell dictionaries. ::
 
-    codespell -L word1,word2,word3,word4
+    codespell -L LIST, --ignore-words-list=LIST
```

Comment:
Maybe turn this into an example like the skip one?

Reply:
Since you mentioned grep(3) man page, I tried to follow its style, which mentions the argument twice (this is why I originally changed it to `-o, --option example,argument` -- to avoid repeating the example argument).

I'm afraid something like
```
codespell -L word1,word2,..., --ignore-words-list=word1,word2,...
```
will look way worse. Or do you mean something different here?

Reply:
Yeah I think I'd agree. I was meaning more like this (rather than dropping the example config entirely):

https://github.com/codespell-project/codespell/blob/eabccd736fb56fb3dc5cc27c219f70f967892a22/README.rst#L58

Reply:
done